commands/daemon.go

```diff
@@ -122,8 +123,23 @@ func getRepo(req *cmds.Request) (repo.Repo, error) {
 	return repo.OpenFSRepo(getRepoDir(req))
 }
 
-func runAPIAndWait(ctx context.Context, node *node.Node, config *config.Config, req *cmds.Request) error {
-	api := impl.New(node)
+func runAPIAndWait(ctx context.Context, nd *node.Node, config *config.Config, req *cmds.Request) error {
+	api := impl.New(nd)
+	blockTime, mineDelay := nd.MiningTimes()
+	blockpAPI := bapi.New(
```

Comment:
boo on not moving this into https://github.com/filecoin-project/go-filecoin/blob/cdf18316b9f65b9d474b56520f944fef4081fcba/node/node.go#L314 but ok